packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -65,7 +65,8 @@ class Scrollable extends StatefulWidget {
   /// ScrollableState scrollable = Scrollable.of(context);
   /// ```
   static ScrollableState of(BuildContext context) {
-    return context.ancestorStateOfType(const TypeMatcher<ScrollableState>());
+    final _ScrollableScope widget = context.inheritFromWidgetOfExactType(_ScrollableScope);
```

Comment:
We could certainly do that. I looked through the existing inherited widget .of()  methods but  I didn't see one that gave you the option of not subscribing.